src/renderer/ScreenController.tsx

```diff
@@ -60,6 +69,7 @@ export default class ScreenController extends Component {
 
   componentDidMount() {
     ipcRenderer.on('error', this.onError)
+    ipcRenderer.on('update-network-status', this.onNetworkChange)
```

Comment:
You could just listen for `DC_EVENT_ERROR_NETWORK` here

Reply:
how would I know when it should be hidden again, then?